manifests/e/errata-ai/Vale/2.24.4/errata-ai.Vale.installer.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: errata-ai.Vale
+PackageVersion: 2.24.4
+InstallerLocale: en-US
+InstallerType: zip
+Installers:
+- Architecture: x64
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: vale.exe
+    PortableCommandAlias: vale
+  InstallerUrl: https://github.com/errata-ai/vale/releases/download/v2.24.4/vale_2.24.4_Windows_64-bit.zip
+  InstallerSha256: 3567E55948C13293853AC83A563D5062F0A975EAC4209FC34F561CC6E4300AF1

```

```suggestion
  InstallerSha256: 3567E55948C13293853AC83A563D5062F0A975EAC4209FC34F561CC6E4300AF1
- Architecture: x86
  NestedInstallerType: portable
  NestedInstallerFiles:
  - RelativeFilePath: vale.exe
    PortableCommandAlias: vale
  InstallerUrl: https://github.com/errata-ai/vale/releases/download/v2.24.4/vale_2.24.4_Windows_386.zip
  InstallerSha256: BD3B9A5AE5D95A055FA68550BB07C120410F0FE2BD27794604356A15B0E01C9F
```
Please add x86 installer